### PR TITLE
Add more examples of different Input Methods

### PR DESCRIPTION
**Pull request by happycoder0011**
Closes # 

## 📝 Description

> Add a brief description
The documentation page just shows examples on text and password as input. It would be a great addition to show more examples such as date, month, time and color picker inputs. I saw reddits and stackoverflow questions asking for date and time picker exclusively not being available in Chakra UI as Material UI and ANTD.
## ⛳️ Current behavior (updates)

> Please describe the current behavior that you are modifying
No Mention of any other input methods supported by Chakra UI Input. Users expect exclusive date time pickers similar to Material UI and ANTD.
## 🚀 New behavior
Added examples on other input methods for the users.

> Please describe the behavior or changes this PR adds

## 💣 Is this a breaking change (Yes/No):
Somewhat - need to install 3rd party libraries for specific components.

### CODE REVIEW

File: content/docs/components/input/usage.mdx [hunk 1]
```diff
@@ -217,3 +217,46 @@ opacity to 1 if you want the placeholder to have a specific color.
...
+### Input Methods other than Text
+Date and Time Picker
+
+```jsx
+  <Input
+              placeHolder="Select Date and Time"
+              size="md"
+              backgroundColor="#ffffff"
+              type="datetime-local"
+            />
```
Comment (noobinthisgame) [inline content/docs/components/input/usage.mdx:230]: I'd go with only this example and add a note about more types and a link to [this list](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input#input_types)

Also please clean up the indentation

Comment (happycoder0011) [reply content/docs/components/input/usage.mdx:230]: ### Input Methods other than Text
You can use different types with `Input` such as dateTime, color, search, file etc. 

Date and Time Picker

```jsx
  <Input
     placeHolder="Select Date and Time"
     size="md"
     backgroundColor="#ffffff"
     type="datetime-local"
    />
```
Check for different input types available : https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input#input_types 

Does this looks good?  Or any other change suggestion?

Comment (noobinthisgame) [reply content/docs/components/input/usage.mdx:230]: Yes that's exactly what I meant :)

**Review by noobinthisgame — APPROVED** [review]
Thanks for adding this @happycoder0011

**Pull request merged**